ci/update-packages.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+######################################################################
+# This script sets up the docker environment for updating packages
+######################################################################
+
+set -e
+set -x
+
+# Check if docker is installed
+if ! [ -x "$(command -v docker)" ]; then
+  echo "Docker is not installed. Please install docker and try again."
+  exit 1
+fi
+
+# Check if we can run docker without sudo
+if ! docker ps > /dev/null 2>&1; then
+  echo "Docker is not running without sudo. Please add your user to the docker group and try again."
+  exit 1
+fi
+
+SOURCE_DIR=$(pwd)/../
+
+TEMP_DIR=$(mktemp -d)
+
+# Copy the current directory to the temp directory
+cp -r ../ $TEMP_DIR
+
+cd $TEMP_DIR
+
+PYTHON_VERSIONS=(3.8 3.9 3.10 3.11)
+
+for PYTHON_VERSION in "${PYTHON_VERSIONS[@]}"
+do
+    # Remove container if it exists
+    docker rm --force flexmeasures-update-packages-$PYTHON_VERSION || true
```

Comment:
Should we check explicitly whether the container exists first? Otherwise this error can happen:

`Error: No such container: flexmeasures-update-packages-3.8`

The script does still continue, but only on docker 20.10.0 and up (https://github.com/docker/cli/issues/2383).